CONTRIBUTING.md

```diff
@@ -39,6 +39,29 @@ You can also use this small bookmarklet tool to fill your Pull Request fields au
 javascript:(function() {var e = document.getElementById('pull_request_body');if (e) {e.value += '### What is this PR for?\nA few sentences describing the overall goals of the pull request\'s commits.\n\n### What type of PR is it?\n[Bug Fix | Improvement | Feature | Documentation | Hot Fix | Refactoring]\n\n### Todos\n* [ ] - Task\n\n### Is there a relevant Jira issue?\n\n### How should this be tested?\nOutline the steps to test the PR here.\n\n### Screenshots (if appropriate)\n\n### Questions:\n* Does the licenses files need update?\n* Is there breaking changes for older versions?\n* Does this needs documentation?';}})();
 ```
 
+## Testing a Pull Request
+You can also test and review a particular Pull Request. Here are two useful ways.
+
+* Using a utility provided from Zeppelin. 
+    
+    ```
+    dev/test_zeppelin_pr.py [# of PR]
+    ```
+
+    For example, if you want to test `#513`, then the command will be:
+
+    ```
+    dev/test_zeppelin_pr.py 513
+    ```
+
+* Another way is using [github/hub](https://github.com/github/hub). 
+     
+    ```
+    hub checkout https://github.com/apache/incubator-zeppelin/pull/[# of PR]
+    ```
+
+The above two methods will help you to test and review the Pull Request.
```

Comment:
@felixcheung I think that "help you to test" is same with "help you test", isn't it? : )